### PR TITLE
Add get_name, get_id for column family handle in C API

### PR DESCRIPTION
**Pull request by muthukrishnan24**

### CODE REVIEW

File: db/c.cc [hunk 1]
```diff
@@ -1054,6 +1054,18 @@ void rocksdb_drop_column_family(
...
 }
 
+uint32_t rocksdb_column_family_handle_get_id(
+    rocksdb_column_family_handle_t* handle) {
+  return handle->rep->GetID();
+}
+
+char* rocksdb_column_family_handle_get_name(
+    rocksdb_column_family_handle_t* handle, size_t* name_len) {
+  auto name = handle->rep->GetName();
```
Comment (ajkr) [inline db/c.cc:1064]: We could save a copy using reference, but it shouldn't matter.

**Review by ajkr — APPROVED** [review]
LGTM, thanks!

**Comment by ajkr** [comment]
"This branch has conflicts that must be resolved"

**Comment by muthukrishnan24** [comment]
@ajkr resolved the merge conflicts. please check.

**Pull request closed without merging**